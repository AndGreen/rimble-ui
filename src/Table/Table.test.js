import Table from './'

describe('Table component sanity', () => {
  it('has name', () => {
    expect(Table.displayName).toBe('Table')
  })

  it('matches default snapshot', () => {
    const component = render(<Table />)
    expect(component).toMatchSnapshot()
  })
})
