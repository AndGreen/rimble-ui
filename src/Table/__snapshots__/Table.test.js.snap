// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component sanity matches default snapshot 1`] = `
<table>
  <thead>
    <tr>
      <th
        colspan="2"
      >
        The table header
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        The table body
      </td>
      <td>
        with two columns
      </td>
    </tr>
  </tbody>
  <tfoot>
    <tr>
      <td
        colspan="2"
      >
        The table footer
      </td>
    </tr>
  </tfoot>
</table>
`;
